tests/util/db.go

```diff
@@ -118,6 +118,23 @@ func CheckSyncState(sourceDB, targetDB *sql.DB, schema string) bool {
 			return false
 		}
 	}
+
+	targetTables, err := dbutil.GetTables(ctx, targetDB, schema)
+	if err != nil {
+		log.Print(err)
+		return false
+	}
+	sourceTableMap := make(map[string]struct{}, len(tables))
+	for _, table := range tables {
+		sourceTableMap[table] = struct{}{}
+	}
+	for _, table := range targetTables {
+		if _, exist := sourceTableMap[table]; !exist {
+			log.Print("The table in target db is not exist in source db:", table)
+			return false
+		}
+	}
```

Comment:
Doesn't the diff check already check this for us?

Reply:
when the table in target database is more than source database, the diff check will not return an error. This pr will fix this.

Reply:
Add a comment to explain this?

Reply:
added